### DISCUSSION TITLE
There's not an easy way to get rid of HELD messages

### DISCUSSION THREAD

**Discussion by teoxrr** [postalserver/postal#2755] [Feature suggestions]
Would be great being able to manage HELD messages, i.e. with a "delete all" option.

**Comment by Pacerino** [comment]
What I always do to cancel the emails is to go into the `postal-server-X` database and edit the entries in the `messages` table. There I set the column `held` to `0` and the column `status` to `HoldCancelled` for the emails I want to cancel. This is a workaround, but it has worked wonderfully so far.

But I agree, a button in the UI would certainly be easier!
